Homework_5.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>The program is actually not smart. You are smart since you compare it with independent results. The program actually gives you a wrong result for large n (and that is not smart). However you could implement something in your code that catches that and then outputs something smart.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>